tools/gather-sysinfo/gather-sysinfo_suite_test.go

```diff
@@ -0,0 +1,13 @@
+package main
```

Comment:
I'm not a fan of using ginkgo for unit tests. We add a great amount of baggage with unclear benefits. What are the benefits of using ginkgo here?

Reply:
Same opinion here, but we decided to use ginkgo for everything.